### ISSUE TITLE
Lack of Linux version of omniorb

### ISSUE THREAD

**Issue by namruf15** [metamorph-inc/omniORB#1]
Hello, could you generate also .whl for Linux platform? Currently on PyPi there're only Windows wheels provided.

**Comment by ksmyth** [comment]
I don't currently have the time to work on this. I'd accept a PR that uses appveyor or travis.

Aside: e.g. `apt-get install python-omniorb` would install https://packages.ubuntu.com/bionic/python-omniorb . Wouldn't that be sufficient?

**Comment by namruf15** [comment]
No, if an user would like to separate functionality from OS dependencies :). Also this solution fits only Ubuntu. Debian i.e. hasn't got version above 4.0.0 inside its distro repos.

**Comment by blancmatter** [comment]
+1 to this request. This would be a fantastic resource.
